src/test/integration/controllers/accounts.controller.e2e-spec.ts

```diff
@@ -0,0 +1,252 @@
+        await app.init();
+    });
+
+    describe('/accounts', () => {
+        it('should return 25 accounts available on blockchain', async () => {
+            await request(app.getHttpServer())
+                .get(`${path}`)
+                .expect(200)
+                .then(res => {
+                    expect(res.body).toHaveLength(25);
+                });
+        });
+
+        it('should return 1 account details', async () => {
+            const params = new URLSearchParams({
+                'size': '1',
+            });
+
+            await request(app.getHttpServer())
+                .get(`${path}?${params}`)
+                .expect(200)
+                .then(res => {
+                    expect(res.body).toHaveLength(1);
+                });
+        });
+
+        it('should return 7 accounts', async () => {
+            const params = new URLSearchParams({
+                'from': '80',
+                'size': '7',
+            });
+
+            await request(app.getHttpServer())
+                .get(`${path}?${params}`)
+                .expect(200)
+                .then(res => {
+                    expect(res.body).toHaveLength(7);
+                });
+        });
+
+        it('should return accounts detailes for a given owner address', async () => {
+            const params = new URLSearchParams({
+                'ownerAddress': 'erd1yj4jlay9rrzahran7jxk89gsg9frxw6l5qyca9dqhp8c4f5e0vdsytwkvl',
+            });
+
+            const expected =
+                [
+                    {
+                        address: "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqhllllsajxzat",
+                        balance: expect.any(String),
+                        nonce: 0,
+                        timestamp: expect.any(Number),
+                        shard: 4294967295,
+                        ownerAddress: "erd1yj4jlay9rrzahran7jxk89gsg9frxw6l5qyca9dqhp8c4f5e0vdsytwkvl",
+                    },
+                ];
+
+            await request(app.getHttpServer())
+                .get(`${path}?${params}`)
+                .expect(200)
+                .then(res => {
+                    expect(res.body).toStrictEqual(expected);
+                });
+        });
+
+        it('should return 400 Bad Request if owner address is not correct', async () => {
+            const params = new URLSearchParams({
+                'ownerAddress': 'erd1yj4jlay9rrzahgsg9frxw6l5qyca9dqhp8c4f5e0vdsytwkvl',
+            });
+            await request(app.getHttpServer())
+                .get(`${path}?${params}`)
+                .expect(400)
+                .then(res => {
+                    expect(res.body.message).toEqual("Validation failed for argument 'ownerAddress' (a bech32 address is expected)");
+                });
+        });
+
+        [
+            {
+                filter: 'sort',
+                value: 'balance',
+            },
+        ].forEach(({ filter, value }) => {
```

Comment:
This test can contain also the below test, no need to create another test just to test another sort type